server/src/controllers/reviewController.js

```diff
@@ -76,6 +78,42 @@ export default class ReviewController {
       .catch(error => handleErrorMessage(res, error));
   }
 
+  /**
+   * getBusinessRating()
+   * @desc retrieves rating for a business
+   * @param {Object} req request object
+   * @param {Object} res response object
+   * @return {Object} message, reviews
+   */
+  static getBusinessRating(req, res) {
+    const { businessId } = req.params;
+
+    const isNotUUID = checkUUID(res, businessId, 'Business');
+    if (isNotUUID) return isNotUUID;
+
+    return Review.findAll({ where: { businessId } })
+      .then((reviews) => {
+        if (reviews === null) {
```

Comment:
I don't think this check here is valid, review will either hold data or be empty, so checking for null is not valid.